tests/test_contract_enforcement.py

```
@@ -0,0 +1,243 @@
+        from agent.skill_registry import SkillRegistry
+        reg = SkillRegistry()
+        with pytest.raises(TypeError, match="unexpected keyword argument"):
+            # type: ignore[arg-type]
+            reg.get(skill_id="local:test", extra=True)
+
+    def test_update_github_token_rejects_unknown(self) -> None:
+        from agent.skill_registry import SkillRegistry
+        reg = SkillRegistry()
+        with pytest.raises(TypeError, match="unexpected keyword argument"):
+            # type: ignore[arg-type]
+            reg.update_github_token(token="ghp_xxx", extra=True)
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "token"

(S106)